### PR TITLE
Stop active support of cross-compilation for Windows

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Updated documentation after removing active support
for MinGW cross-compilation

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -12,6 +12,17 @@ openssl<sup>1</sup> | >=v1.x
 <sup>1</sup> openssl is needed if libdrizzle-redux is compiled with support for
 SSL connections.
 
+As mentioned in the release notes for [v5.5.3](https://github.com/sociomantic-tsunami/libdrizzle-redux/releases/tag/v5.5.3)_ the MinGW is broken.
```
Comment (mathias-lang-sociomantic) [inline RELEASE_NOTES.md]: the MinGW *build* is broken

**Review by mathias-lang-sociomantic — CHANGES_REQUESTED** [review]
Just a typo, oLGTM

**Comment by andreas-bok-sociomantic** [comment]
ping @mathias-lang-sociomantic

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -12,6 +12,17 @@ openssl<sup>1</sup> | >=v1.x
...
 
+As mentioned in the release notes for [v5.5.3](https://github.com/sociomantic-tsunami/libdrizzle-redux/releases/tag/v5.5.3)_ the MinGW build is broken.
+
+Given the limited resources of the contributors we have decided not to fix this
+and instead stop active support for building for Windows (cross-compile) using
+MinGW.
+
+However, at the moment there is no plans to remove the existing support for
+MinGW from the codebase.
+Thus, should any contributers wish to add MinGW support again, they are more
```
Comment (mathias-lang-sociomantic) [inline RELEASE_NOTES.md]: `contributors`

Comment (andreas-bok-sociomantic) [reply RELEASE_NOTES.md]: Damn it

**Comment by andreas-bok-sociomantic** [comment]
Fixed typo

**Review by mathias-lang-sociomantic — APPROVED** [review]

**Pull request merged**